### ISSUE TITLE
I don't know why but my contribution count got reduced automatically.

### ISSUE THREAD

**Issue by Dhiraj240** [isaacs/github#1400]
Yesterday, my contribution counts on my github profile was 186 but today it went down to 176.May i know what all could be the possible situations.
I am totally clueless about it.

**Comment by Dhiraj240** [comment]
One point to note is that i haven't touched my single repository i.e neither admitted nor deleted anything which could cause this probable decrease.

**Comment by YSavir** [comment]
Did you happen to remove an email from github? I recently changed mine and saw a change, as github compares your emails to the commit author emails to see if they should be attributed to you.

**Comment by Dhiraj240** [comment]
Nope, i did nothing related to that, right now i am having my university exams so i stopped my code contribution in opensource also didn't even touch the single bit of my profile.It happened within this week.

**Comment by clarkbw** [comment]
You could contact support about this and they can investigate https://github.com/contact

**Comment by Dhiraj240** [comment]
Okay, i have to it probably.

**Comment by Rudy-Zidan** [comment]
Hi, I just faced the same issue anyone know why?

**Comment by oprogramador** [comment]
@Dhiraj240 

It may be because it shows the number of contributions for the past 12 months. So it could sometimes increase and sometimes decrease.

**Comment by charleslukes** [comment]
I noticed this too just now, I added a contributor to a project which is just a fun project, he only reviews my code(It's been like two months already). I decided to merge my branch to master today but all of a sudden my commits required. I am totally confused about how and why???

**Comment by amreshk005** [comment]
> Did you happen to remove an email from github? I recently changed mine and saw a change, as github compares your emails to the commit author emails to see if they should be attributed to you.

Yes! I changed my email. Can I get it back

**Comment by mgruhler** [comment]
Yes, you need to have the "old" email address still associated with your account. This is add it to your emails, but you don't need to validate it. See also [this](https://help.github.com/en/articles/changing-your-github-username#your-git-commits) and [this](https://help.github.com/en/articles/setting-your-commit-email-address) help pages.

**Comment by amreshk005** [comment]
> Yes, you need to have the "old" email address still associated with your account. This is: add it to your emails, but you don't need to validate it. See also [this](https://help.github.com/en/articles/changing-your-github-username#your-git-commits) and [this](https://help.github.com/en/articles/setting-your-commit-email-address) help page

Yeah! It worked

**Comment by HeavyMetalCookies** [comment]
I've been lifting weights every time I make a git commit. Usually twice a day. So I noticed when all of my contributions got doubled for some reason today. Guess it wasn't the best way to track my exercise and productivity at the same time.

**I am guessing the contribution size is somehow scaled relative to the size of other commits.**

**Comment by Akshit6828** [comment]
IT also happened to me just now. Can anyone tell me? I have my GitHub readme stats and it showed me one less than it was having in morning,
